capstone/src/main/java/com/google/sps/servlets/BlobstoreKeyServlet.java

```diff
@@ -1,20 +1,23 @@
+package com.google.sps.servlets;
+
 import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+
 import com.google.appengine.api.blobstore.BlobKey;
 import com.google.appengine.api.blobstore.BlobstoreService;
 import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
-import javax.servlet.annotation.WebServlet;
 
-@WebServlet("serve")
+@WebServlet("blobstore-serve")
```

Comment:
It's possible that your PR is just outdated and needs to pull from master, however in case you made these changes I'd double check with Anika to make sure there aren't any other locations you need to update this.